pyrobuf/parse_proto.py

```diff
@@ -65,7 +65,7 @@ class Parser(object):
 
     # These tokens are parsed by the parser but are not supported by the
     # code that build the C extension files and definitions
-    unsupported_tokens = ('MAP_FIELD', 'ONEOF',)
+    unsupported_tokens = ('MAP_FIELD',)
```

Comment:
Maybe make this dependent on proto syntax, so that `ONEOF` is still marked as "unsupported" for proto2?